org.opentosca.bus.management.plugins.remote.service.impl/src/org/opentosca/bus/management/plugins/remote/service/impl/ManagementBusPluginRemoteServiceImpl.java

```diff
@@ -649,7 +649,14 @@ private String createParamsString(final Object params, final String publicIP) {
                 final String replacedValue = param.getValue().replaceFirst("container", publicIP);

```

Comment:
Strange that we need this adaption at all. If you configure and start the container correctly this shouldn't be necessary, right?